apps/preview/nuxt/pages/showcases/Link/NuxtLink.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="space-x-4">
-    <SfLink :tag="NuxtLink" to=""> NuxtLink </SfLink>
+    <SfLink :tag="NuxtLink" :to="{ path: '#', query: { docs: true } }"> NuxtLink </SfLink>
```

Comment:
Our example rely on `docs=true` query param. If I wouldn't have added it here, click on the link would open side navigation within the iframe.

Reply:
I understand why its happening, so maybe better solution would be 

```
<template>
  <div class="space-x-4">
    <SfLink :tag="NuxtLink" :to="{ path: '#', query: currentRoute }"> NuxtLink </SfLink>
  </div>
</template>

<script lang="ts" setup>
import { resolveComponent } from 'vue';
import { SfLink } from '@storefront-ui/vue';
import { useRouter } from 'vue-router';

const route = useRouter();
const currentRoute = route.currentRoute.value.query;
const NuxtLink = resolveComponent('NuxtLink');
</script>
```
to remember original query and just apply it always so it would not be reseted ? 🤷

Reply:
It would add extra complexity to the example, and I think it'd be hard to understand why we do it. I'd not add it, as it suppose to be an easy example showing how you can use SfLink with the NuxtLink.

Reply:
but having `docs: true` is also confusing, but still better then my solution

Reply:
Yeah, I think we may change the way we disable side navigation to something else. Maybe a prop or something more static, as it would not change anyways.

Reply:
not sure how we can inject otherwise data into iframe so framework would be able to read it